charts/simcore-charts/resource-usage-tracker/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "resource-usage-tracker.fullname" . }}-test-connection"
+  labels:
+    {{- include "resource-usage-tracker.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "resource-usage-tracker.fullname" . }}:{{ .Values.service.port }}']
+  restartPolicy: Never
```

Comment:
Q: what is the concept here? does that mean that if your busybox would break, this would not be restarted? or does that mean if RUT does not answer you do not restart it?

Reply:
I am also curious

Reply:
This acts like unit tests. It does not affect any running "services". One can run these tests on demand to ensure that everything is configure properly.